Pipfile

```diff
@@ -1,19 +1,27 @@
 [[source]]
+
 url = "https://pypi.python.org/simple"
 verify_ssl = true
 name = "pypi"
 
+
 [packages]
+
 twine = "*"
 Sphinx = "*"
 "yolk3k" = "*"
 pylint = "*"
 pandas = "*"
 requests = "*"
 protobuf = "*"
+pytest = "*"
```

Comment:
Could pytest and pylint be part of dev-packages also?

Reply:
yes, twine and yolk aswell i guess